workflow-ui/core-common/src/main/java/com/squareup/workflow1/ui/container/BackStackScreen.kt

```diff
@@ -115,6 +115,13 @@ public operator fun <T : Screen> BackStackScreen<T>.plus(
   return other?.let { BackStackScreen(frames + it.frames) } ?: this
 }
 
+@WorkflowUiExperimentalApi
+public operator fun <T : Screen> BackStackScreen<T>.plus(
+  other: BackStackScreen<T>?
+): BackStackScreen<T> {
+  return other?.let { BackStackScreen(frames + it.frames) } ?: this
+}
```

Comment:
Could separate this out to a different PR that's not tutorial related. but also 🤷🏻

Reply:
Yeah, this seems like it leaked in from another PR (that I thought I had already merged).